### ISSUE TITLE
No Module named 'python_passsword'

### ISSUE THREAD

**Issue by Akul2010** [AnonymousX86/Python-Password#8]
When I run the program with the command `python PyPassword.py`, it gives an error saying `No Module named python_password`. I am using vscode, which shouldn't make a difference.

1. Click on save and run.
2. Or click on save and then put in the terminal, python PyPassword.py

I had expected to see the password manager.

**Screenshots**
![image](https://user-images.githubusercontent.com/75178259/147836541-c03c4882-ef62-4686-ba27-1a7e2946fbda.png)

**Version**
v'0.2.5'

**Comment by grammyy** [comment]
Have you tried running "py setup.py"?

**Comment by Akul2010** [comment]
Yes, but all it did was give me a list of commands, so I tried `py setup.py build`, but it didn't do anything.
